### DISCUSSION TITLE
Multiple tables on one page

### DISCUSSION THREAD

**Discussion by vv-ac** [filamentphp/filament#2329] [Help] [answered]
Hi all,

I use mutlple forms on one custome page which works just fine.
Now, I would like to use multiple tables on the same page, too (partially hidden in modals).

Is that possible?

Thanks

**Comment by danharrin** [comment, marked as answer]
Each table needs to be in its own Livewire component, so you could nest them within your custom page view?

**Reply by vv-ac** [reply]
Thanks for the fast response.

So, you mean creating a new component (incl. a view) and then do a blade include for the view to render the table?

**Reply by danharrin** [reply]
Create a new Livewire component, and then render the Livewire component in the view, not just a blade include. `@livewire(...)`

**Reply by vv-ac** [reply]
Thanks, this is likely to work. I did not test it but go another way than having multiple tables.

**Reply by khanakia** [reply]
@riskmate-ac can you share with me some example of how can i include multiple forms on one page? I could not find any solution.

**Reply by vv-ac** [reply]
Hi @khanakia , you can find those instructions here: https://filamentphp.com/docs/2.x/forms/getting-started#using-multiple-forms
In simple words: Use different names and corresponding methods. Once you have set up the first page with two forms it's easy to reproduce that a thousand times.
